docs/cli.md

```diff
@@ -67,18 +68,18 @@ Examples:
+| Flag                                                         | Description                                                                                                                                                                                                                                                          |
+|--------------------------------------------------------------|----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| &minus;&minus;kind &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; | Clean resources for all runs of a specific kind (e.g. job).                                                                                                                                                                                                          |
+| &minus;&minus;id &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp;   | Delete the resources of the mlrun object twith this identifier. For most function runtimes, runtime resources are per Run, and the identifier is the Run’s UID. For DASK runtime, the runtime resources are per Function, and the identifier is the Function’s name. |
 
   
-Options| Description |
-| ----------- | ----------- |
-|  &minus;&minus;api |  URL of the mlrun-api service.|
-|  -ls, &minus;&minus;label-selector &nbsp; &nbsp; &nbsp;&nbsp; &nbsp; &nbsp;&nbsp; &nbsp; &nbsp;| Delete only runtime resources matching the label selector.|
-|  -f, &minus;&minus;force   |  Delete the runtime resource even if they're not in terminal state or if the grace period didn’t pass.|
-|  -gp, &minus;&minus;grace-period &nbsp; &nbsp; &nbsp;  &nbsp; &nbsp; &nbsp; | Grace period, in seconds, given to the runtime resource before they are actually removed, counted from the moment they moved to the terminal state.  |
+| Options                                       | Description                                                                                                                                         |
+|-----------------------------------------------|-----------------------------------------------------------------------------------------------------------------------------------------------------|
+| &minus;&minus;api                             | URL of the mlrun-api service.                                                                                                                       |
+| &minus;ls, &minus;&minus;label&minus;selector | Delete only runtime resources matching the label selector.                                                                                          |
```

Comment:
| &minus;ls, &minus;&minus;label&minus;selector | Delete only the runtime resources matching the label selector.